.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 💬 Questions / Help
+labels: ':speech_balloon: Question'
+about: If you have questions, please check StackOverflow
+---
+
+## 💬 Questions and Help
+
+### Please note that this issue tracker is not a help forum and this issue will be closed.
+
+For questions or help, you can also check on StackOverflow https://stackoverflow.com/questions/tagged/jest-preset-angular) tag on [StackOverflow](https://stackoverflow.com/questions/ask
```

Comment:
I gonna correct this to comply with markdown syntax and rebase master then it’s ready to merge